.github/workflows/release.yml

```diff
@@ -370,3 +370,19 @@ jobs:
         asset_path: build\\WasmEdge-${{ needs.create_release.outputs.version }}-Windows.zip
         asset_name: WasmEdge-${{ needs.create_release.outputs.version }}-windows.zip
         asset_content_type: application/zip
+
+    - name: Package Windows Installer
+      run: |
+        $Env:product_version = "${{ needs.create_release.outputs.version }}"
+        . "$Env:WIX\bin\candle.exe" -arch x64 -o build\wasmedge.wixobj .github\scripts\wasmedge.wxs
+        . "$Env:WIX\bin\light.exe" -out build\WasmEdge-${{ needs.create_release.outputs.version }}-windows.msi build\wasmedge.wixobj
```

Comment:
Should this version be the same as the `env.product_version` in `build.yml`.

Reply:
In the previous discussion, we find the format is `x.y.z.build`. If we only use the release tag `create_release.outputs.version`, it will be `x.y.z` only. Is this acceptable?

Reply:
According to the Windows document, only the first three fields of version number would be actually used. So if you installed the `0.9.0` version and then install `0.9.0.1` or other versions, you will see those versions coexist, and later installed versions will replace old files. Only `x.y.z` has been changed, then the installer will remove all the old versions and install the new version. So I think it's acceptable. @hydai

Reply:
Great. Then, I am merging this.